README.md

```diff
@@ -1,2 +1,34 @@
 # openshift-launchpad
 A project template to accelerate onboarding to the OpenShift platform
+
+## Table of Contents
+
+1. [Getting Help or Reporting an Issue](#getting-help-or-reporting-an-issue)
+1. [How to Contribute](#how-to-contribute)
+1. [License](#license)
+
+## Getting Help or Reporting an Issue
+
+To report bugs/issues/features requests, please file an [issue](https://github.com/bcgov/openshift-launchpad/issues).
+
+## How to Contribute
+
+If you would like to contribute, please see our [contributing](CONTRIBUTING.md) guidelines.
+
+Please note that this project is released with a [Contributor Code of Conduct](CODE-OF-CONDUCT.md). By participating in this project you agree to abide by its terms.
+
+## License
```

Comment:
I would suggest we make a LICENSE file and link it here. That way our README is smaller and less intimidating. Thoughts?

Reply:
We also have a license file. I'm not sure why other teams also include this note in their main README, but from looking around it seems pretty consistent? So I just did the same.

It might come from here, originally?
https://developer.gov.bc.ca/Sample-Readme-File

Reply:
Oh shoot, yeah then I guess it makes sense to keep it like this for simplicity sake